README.md

```diff
@@ -235,10 +237,11 @@ gateway-conformance test --specs trustless-gateway,-trustless-gateway-ipns
 
 ### Skip a specific test
```

Comment:
Could  we add `### Run only a specific test` with example that uses go's `go test -run <pattern>` and use the same pattern as in the skip example?